python/src/aiconfig/editor/server/server.py

```diff
@@ -177,8 +180,14 @@ async def run() -> FlaskResponse:
     state = get_server_state(app)
     aiconfig = state.aiconfig
     request_json = request.get_json()
+    cancellation_token_id: Union[str, None] = None
```

Comment:
nit: str | None or the older Optional[str] is ok. Union[str, None] is nonstandard

Reply:
Fixed